packages/suite/src/components/suite/NavigationBar/index.tsx

```diff
@@ -55,6 +57,32 @@ export const NavigationBar: React.FC = () => {
         setOpened(false);
     };
 
+    let sessionCount = 0;
+    coinjoinAccounts.forEach(({ session }) => {
```

Comment:
I don't get this, we are supposed to have always only 1 coinjoin account, right?

Reply:
Yes, per device. It is supported by the coinjoin package to theoretically run > 1 sessions, 1 per device. But it will be disabled with the release version on CJ.